CONTRIBUTING.md

```diff
@@ -1,6 +1,42 @@
 # Contribute to Personium
 
+Thank you for taking your time and considering for contribution to Personium Project.
```

Comment:
Just a comment. Other OSS project's CONTRIBUTING.md is helpful.
https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md

Reply:
Thanks. I agree.
I saw several other projects' CONTRIBUTING.md files and wrote a simplest version this time.
But I haven't check the one from elastic search.
We should keep improving this file.